library/webpack-loaders/react-universal-server-loader.js

```diff
@@ -7,18 +7,18 @@ function ReactUniversalServerLoader(source, map) {
   const filename = relative(this.options.context, this.resourcePath)
   const importPath = relative(this.context, this.resourcePath)
   const id = filename.replace(/[/\.]/g, '_')
-  return createServerCode({ importPath, scriptSrc: '/' + filename, id })
+  return createServerCode({ importPath, scriptSrc: filename, id })
 }
 
 function createServerCode({ importPath, scriptSrc, id }) {
   const appScript = isProduction
-    ? `<script defer src='${scriptSrc}?v=${Date.now()}'></script>`
+    ? `<script defer src={'/${scriptSrc}?v=${Date.now()}' }></script>`
```

Comment:
This whole if-statement becomes obsolete when we switch to hashes for these chunks.

Reply:
problem is probably that we don't know the chunk hash (filename) at this point, but we could probably introduce a variable for that.

We don't want to move the script tag to the head because then we would have to add code that wait's for the dom node to be available.

Reply:
Hmm, I should test it, MDN states:
> This Boolean attribute is set to indicate to a browser that the script is meant to be executed after the document has been parsed, but before firing DOMContentLoaded.

So is 'parsed' enough to call `document.getElementById('${id}')`?